### PR TITLE
Add ex 02

### PR DESCRIPTION
**Pull request by VoLeQuocHoang**
Optional messages

### CODE REVIEW

File: NetExercise/Basic/Exercise02.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+namespace NetExercise.Basic
+{
+    public class Exercise02
+    {
+        //(bai 2)Tinh tong cac chu so cua 1 so Nguyen
+        public static int TongSoNguyen(int a)
+        {
+            int tong = 0;
```
Comment (voquanghoa) [inline NetExercise/Basic/Exercise02.cs:8]: Use `var` instead

Comment (voquanghoa) [inline NetExercise/Basic/Exercise02.cs:8]: Try to use `English`

Comment (VoLeQuocHoang) [reply NetExercise/Basic/Exercise02.cs]: Done.

Comment (voquanghoa) [reply NetExercise/Basic/Exercise02.cs]: Rename unit test name too

Comment (VoLeQuocHoang) [reply NetExercise/Basic/Exercise02.cs]: Ok done.

**Pull request merged**